crawl-ref/source/dat/des/branches/crypt.des

```diff
@@ -1463,6 +1486,884 @@ cG.MGM.GcG.1.GcG.1.GcG.1.Gc
+NSUBST: - = @ / -
+SUBST:  ' = x:40 ., ` = x:70 ., - = x:100 .
+MAP
+xxxx-x-x-x-xxxx
+xxx'.`.'.`.'xxx
+xxxx...'...xxxx
+x'xx`.`x`.`xx'x
+-..`.......`..-
+x`....`.`....`x
+-..`.`x-x`.`..-
+x''x..-x-..x''x
+-..`.`x-x`.`..-
+x`....`.`....`x
+-..`.......`..-
+x'xx`.`x`.`xx'x
+xxxx...'...xxxx
+xxx'.`.'.`.'xxx
+xxxx-x-x-x-xxxx
+ENDMAP
+
+NAME:    nicolae_crypt_ferns_of_the_dead
+TAGS:    transparent
+DEPTH:   Crypt
+KITEM:   L = % w:30 / * / nothing w:40
+KMONS:   L = 0 w:30 / 9 / nothing w:40
+SHUFFLE: DEFH
+SUBST:   ' = x:40 ., ` = x:60 ., - = x:80 ., ! = x:100 .
+SUBST:   D = @, E : x@, F : xx@, H : xxxx@
+MAP
+xxxxxxxxxEExxxxxxxxx
+xL'L`L-L!..-L`L'..Lx
+x.'.`.-.!..-.`.'.''x
+x...'.`.-..`.'....Lx
+x''.'.`.-..`.'.''``x
+xL....'.`..'......Lx
+x``''.'.`..'.''``--x
+xL......'.........Lx
+x--``''.'..''``--!!x
+D..................F
+D..................F
+x!!--``''..'.''``--x
+xL.........'......Lx
+x--``''.'..`.'.''``x
+xL......'..`.'....Lx
+x``''.'.`..-.`.'.''x
+xL....'.`..-.`.'...x
+x''.'.`.-..!.-.`.'.x
+xL..'L`L-..!L-L`L'Lx
+xxxxxxxxxHHxxxxxxxxx
+ENDMAP
+
+NAME:    nicolae_crypt_for_the_love_of_god_montresor
+TAGS:    transparent
+DEPTH:   Crypt
+: kitem("- = " .. dgn.loot_potions)
+KITEM:   ` = stone / stone q:2 / stone q:3 / stone q:4 / nothing w:40, \
+             any potion / nothing w:100
+KMONS:   p = human spectre
+KMONS:   ` = 0 w:40 / 9 / nothing w:200
+KFEAT:   P = x
+SHUFFLE: Pp-/Qq!, p-, Rr/Ss
+SUBST:   R = @, S = x, r = ., Qqs! = `
+KMASK:   P = no_wall_fixup
+KPROP:   p- = no_tele_into
+KMASK:   p- = opaque
+KMASK:   p- = no_monster_gen
+KMASK:   p- = no_trap_gen
+KMASK:   p- = no_item_gen
+: dgn.delayed_decay(_G, 'p', 'human corpse')
+MAP
+xxxxxxxxxxxxxx
+xxx``xp-xq!xxx
+xxx``xPPxQQxxx
+x``........``x
+x``........``x
+xxx..GxxG..xxx
+x``..xxxx..``x
+x``..xxxx..``x
+xxx..GxxG..xxx
+Rrr........``x
+Rrr........``x
+xxxssx``x``xxx
+xxxssx``x``xxx
+xxxSSxxxxxxxxx
+ENDMAP
+
+NAME:   nicolae_crypt_fracryptal
+TAGS:   transparent
+DEPTH:  Crypt
+NSUBST: Q = @ / x
+SUBST:  - = %0...., x = xxxxxxx., % = %%%*, 0 = 0009, * = ***|, 9 = 8889
+MAP
+xxxxxxxQxxxxxxxQxxxxxxxQxxxxxxx
+x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x
+x.....x.x.....x.x.....x.x.....x
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+Q-...........-x.x-...........-Q
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+x.....x.x.....x.x.....x.x.....x
+x-x-x-x.x-x-x-x.x-x-x-x.x-x-x-x
+xxxxxxx.xxxxxxx.xxxxxxx.xxxxxxx
+Q-...........................-Q
+xxxxxxx.xxxxxxx.xxxxxxx.xxxxxxx
+x-x-x-x.x-x-x-x.x-x-x-x.x-x-x-x
+x.....x.x.....x.x.....x.x.....x
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+Q-...........-x.x-...........-Q
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+x.....x.x.....x.x.....x.x.....x
+x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x
+xxxxxxxQxxxxxxxQxxxxxxxQxxxxxxx
+ENDMAP
+
+NAME:   nicolae_crypt_golgotha
+TAGS:   transparent
+DEPTH:  Crypt
+KMONS:  S = curse skull
+KMONS:  s = flying skull
+SUBST:  S = SSsss, s = sss.
```

Comment:
I know NSUBST isn't fully embracing randomness or the desire to just kill players, but the higher end of zero to five curse skulls in an open space is kind of obscenely threatening for most characters, and therefore I'd only accept this with some numeration for such.